src/static/css/pages/sub-pages.less

```diff
@@ -3,10 +3,51 @@
    Sub-Pages pages at /sub-pages/*.
    ========================================================================== */
 
+.green-bullets {
+    li {
+        color: @black;
+        list-style: none;
+    }
+
+    li:before {
+        position: relative;
+        top: .1em;
+        margin-left: -1em;
+        padding-right: .3em;
+
+        color: @green;
+        //Square bullet point
+        content: "▪ ";
+        font-size: 1.5em;
+
+
+    }
+}
+
+
 .sub-page {
     .webfont-regular();
+
+    a[href $='.pdf'] {
```

Comment:
That's a good point. I'll just note that we extensively tested to see if there was a way to do this without additional markup that
1. Did not underline the icon, and
2. Had the icon break appropriately at ends of lines in all scenarios.

It may be that it's better to sacrifice the first thing so that content editors don't have to worry about the markup to get the icon, but you might want to discuss with design/UX if they're cool with that tradeoff.